### DISCUSSION TITLE
Welcome to OpenScanCloud Discussions!

### DISCUSSION THREAD

**Discussion by OpenScanEu** [OpenScanEu/OpenScanCloud#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by michelsaiful-esimple** [comment]
Hi, the project is super interesting. Which kind of photogrammetry software are you using on server side? Will it ever be released as open to install in local machines?
